style.css

```diff
@@ -0,0 +1,106 @@
+/* ------Global Config------ */
+*,
+::after,
+::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* ------Body------ */
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 62.5%;
+}
+
+/* ------main------ */
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+/* ------Header------ */
+#header {
+  display: flex;
+  flex-direction: column;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 24px 24px;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.logotext {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  text-decoration: none;
+  line-height: 20px;
+  font-style: normal;
+}
+
+/* ------Headline------ */
+#headline {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./images/header-shapes\ mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
Your design is missing the lower background image and design , please it is really important you follow design templates to make your design attractive to recruiters 😄 

 Figma design :

![image](https://user-images.githubusercontent.com/20537712/235182445-b40c1634-555e-457e-ae2c-13f2cbc3b426.png)

Your design :

![image](https://user-images.githubusercontent.com/20537712/235182613-90e2092a-d7d7-4009-a3ad-856f74f811f6.png)